configs/icds-14lakh_reduced.yml

```diff
@@ -264,7 +264,7 @@ services:
     # this should have standbys for HA
     # need higher performing SSDs
     usage_capacity_per_node: 25000
-    include_ha_resources: True
+    # include_ha_resources: True
```

Comment:
@snopoke I assumed that this was for the DR. Is that correct? I did not include any HA resource in any of the spreadsheet